### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Dine16**
add header.php

### CODE REVIEW

**Comment by LouisBruge** [comment]
Can you add the collapse feature of the library Bootstrap

**Comment by Dine16** [comment]
add the burger menu code

File: src/html/header.php [hunk 1]
```diff
@@ -4,10 +4,21 @@
...
+                            <span class="icon-bar"></span>
+                        </button>
+                        <!-- End burger menu -->
+                        <!-- Logo -->
                         <a class="navbar-brand" href="#">
                             <img src="http://via.placeholder.com/70x70" alt="logo">
                         </a>
                     </div>
+                        <!-- End logo -->
+
```
Comment (LouisBruge) [inline src/html/header.php]: Remove this line. It's useless. 
It will be write into the main pages

File: src/html/header.php [hunk 2]
```diff
@@ -4,10 +4,21 @@
             <nav class="navbar navbar-default">
                 <div class="container-fluid">
                     <div class="navbar-header">
+                        <!-- Burger menu -->
+                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" id="bs-example-navbar-collapse-1 aria-expanded="false">
```
Comment (LouisBruge) [inline src/html/header.php]: please, change the
```html
<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" id="bs-example-navbar-collapse-1 aria-expanded="false">
```
into a
```html
<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#top-navbar" aria-expanded="false">
```

File: src/html/header.php [hunk 3]
```diff
@@ -4,10 +4,21 @@
...
+                        </button>
+                        <!-- End burger menu -->
+                        <!-- Logo -->
                         <a class="navbar-brand" href="#">
                             <img src="http://via.placeholder.com/70x70" alt="logo">
                         </a>
                     </div>
+                        <!-- End logo -->
+                    
                     <ul class="nav navbar-nav navbar-right">
```
Comment (LouisBruge) [inline src/html/header.php]: before the 
```html
<ul class="nav navbar-nav navbar-right">
```
can you add a div tag : 
```html
<div class="collapse navbar-collapse" id="top-navbar">
```

and close the tag, between the lines 26 and 27, with a close </div> tag

**Review by LouisBruge — CHANGES_REQUESTED** [review]
Some modifications before to merge your work

**Comment by Dine16** [comment]
add footer.php

**Comment by Dine16** [comment]
just made the modifications needed!

**Pull request merged**